waku/v2/node/rest/store/openapi.yaml

```
@@ -0,0 +1,203 @@
+            only WakuMessages that are linked to any of the given
+            content topics will be delivered in the get response.
+            It should be a URL-encoded-comma-separated string.
+          example: 'my%20first%20content%20topic%2Cmy%20second%20content%20topic%2Cmy%20third%20content%20topic'
+
+        - name: startTime
+          in: query
+          schema:
+            type: string
+          description: >
+            The inclusive lower bound on the timestamp of
+            queried WakuMessages. This field holds the 
+            Unix epoch time in nanoseconds as a 64-bits 
+            integer value.
+          example: '1680590945000000000'
+
+        - name: endTime
+          in: query
+          schema:
+            type: string
+          description: >
+            The inclusive upper bound on the timestamp of 
+            queried WakuMessages. This field holds the 
+            Unix epoch time in nanoseconds as a 64-bits 
+            integer value.
+          example: '1680590945000000000'
+
+        - name: senderTime
+          in: query
+          schema:
+            type: string
+          description: >
+            Cursor field intended for pagination purposes.
+            Represents the Unix time in nanoseconds at which a message was generated.
+            It could be empty for retrieving the first page,
+            and will be returned from the GET response so that
+            it can be part of the next page request.
+          example: '1680590947000000000'
+
+        - name: storeTime
+          in: query
+          schema:
+            type: string
+          description: >
+            Cursor field intended for pagination purposes.
+            Represents the Unix time in nanoseconds at which a message was stored.
+            It could be empty for retrieving the first page,
+            and will be returned from the GET response so that
+            it can be part of the next page request.
+          example: '1680590945000000000'
+
+        - name: digest
+          in: query
+          schema:
+            type: string
+          description: >
+            Cursor field intended for pagination purposes.
+            URL-base64-encoded string computed as a hash of the 
+            a message content topic plus a message payload.
+            It could be empty for retrieving the first page,
+            and will be returned from the GET response so that
+            it can be part of the next page request.
+          example: 'Gc4ACThW5t2QQO82huq3WnDv%2FapPPJpD%2FwJfxDxAnR0%3D'
+
+        - name: pageSize
+          in: query
+          schema:
+            type: string
+          description: >
+            Number of messages to retrieve per page
+          example: '5'
+
+        - name: ascending
+          in: query
+          schema:
+            type: string
+          description: >
+            "1" for paging forward, "0" for paging backward
+          example: '1'
```

I am not convinced with the `1`. In my opinion this should be a boolean: `true` or `false`.

From the OpenAPI docs: https://swagger.io/docs/specification/data-models/data-types/#boolean

> type: boolean represents two values: `true` and `false`. Note that truthy and falsy values such as `"true"`, `""`, `0` or `null` are not considered boolean values.


```suggestion
        - name: ascending
          in: query
          schema:
            type: boolean
          description: >
            "true" for paging forward, "false" for paging backward
          example: true
```

---

For now I'd use 'string' as per https://github.com/waku-org/nwaku/pull/1630#discussion_r1159365472